<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentMain</class>
 <widget class="QMainWindow" name="StudentMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>609</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>考试系统客户端</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: rgb(232, 232, 232);
}
QRadioButton {
	background-color:white;
	padding:0px 20px;
}
QCheckBox {
	background-color:white;
	padding:0px 20px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="nextProblem">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>510</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #1abc9c;
	border: none;
	color:white;
	font-size: 16px;
	border-radius: 6px;
}
QPushButton:hover {
	background-color: rgb(47,226,191);
}
</string>
    </property>
    <property name="text">
     <string>下一题</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>510</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #e74c3c;
	border: none;
	color:white;
	font-size: 16px;
	border-radius: 6px;
}
QPushButton:hover {
	background-color: #ec7063;
}
</string>
    </property>
    <property name="text">
     <string>交 卷</string>
    </property>
   </widget>
   <widget class="QLabel" name="examName">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>691</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="examTime">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>101</width>
      <height>570</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
   </widget>
   <widget class="QLabel" name="wait">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>100</width>
      <height>160</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="r/r.qrc">:/clock.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>800</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>630</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(249, 249, 250);
padding-left:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>580</y>
      <width>500</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mask">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>631</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="o1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>630</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="o2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>630</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="o3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>630</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="o4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>630</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="c1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>630</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="c2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>630</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="c3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>630</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="c4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>630</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>mask</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>nextProblem</zorder>
   <zorder>submit</zorder>
   <zorder>examName</zorder>
   <zorder>examTime</zorder>
   <zorder>listWidget</zorder>
   <zorder>line</zorder>
   <zorder>title</zorder>
   <zorder>status</zorder>
   <zorder>wait</zorder>
  </widget>
 </widget>
 <resources>
  <include location="r/r.qrc"/>
 </resources>
 <connections/>
</ui>
